### ISSUE TITLE
The method 'toImage' isn't defined for the class 'ContainerLayer' after upgrading to Flutter 1.9

### ISSUE THREAD

**Issue by rasmusir** [appspector/flutter-plugin#13]
I'm getting the following error while trying to compile my project uysing Appspector 0.0.7 with Flutter 1.9.1+hotfix.2, dart 2.5.0

```
Compiler message:
../../software/flutter/.pub-cache/hosted/pub.dartlang.org/appspector-0.0.7/lib/src/request_receiver.dart:39:8: Error: The method 'toImage' isn't defined for the class 'ContainerLayer'.
 - 'ContainerLayer' is from 'package:flutter/src/rendering/layer.dart' ('../../software/flutter/packages/flutter/lib/src/rendering/layer.dart').
Try correcting the name to the name of an existing method, or defining a method named 'toImage'.
      .toImage(renderObject.paintBounds, pixelRatio: ratio > 1.0 ? 1.0 : ratio);
       ^^^^^^^
Compiler failed on /home/<me>/GitHub/flutter_kotlin_test/lib/main.inquinova.dart

FAILURE: Build failed with an exception.

* Where:
Script '/home/<me>/software/flutter/packages/flutter_tools/gradle/flutter.gradle' line: 765

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebugArm64'.
```

As far as I can tell the method toImage still exist on ContainerLayer, but something seems wonky.

**Comment by Dimdron** [comment]
Hi @rasmusir 
Thanks for your report! We'll check it

**Comment by rasmusir** [comment]
@Dimdron I added a fix for it as a PR, not sure where to put it so I pointed it towards your master branch.

**Comment by Dimdron** [comment]
Hey, @rasmusir thanks a lot!  Looks like it's working properly!

**Comment by Dimdron** [comment]
Did you experience any other issues?

**Comment by rasmusir** [comment]
Well, screenshots won't really show up in the web, not sure if it's an implementation problem on the Flutter side, Plugin side or the Web side. But at least this allows the package to be built :)

**Comment by Dimdron** [comment]
Yeah, I found this issue. We're using method channel from a background thread on Android side. It causes troubles with some monitors.

**Comment by Dimdron** [comment]
Flutter updates produce a lot of compatibility issues. 😕

**Comment by Dimdron** [comment]
Hey, @rasmusir we've just released 0.0.8 version. Could you try it?

**Comment by rasmusir** [comment]
It's working fine!
